NeuroID/src/main/java/com/neuroid/tracker/NeuroIDPublic.kt

```diff
@@ -0,0 +1,189 @@
+
+    /**
+     * Set a dev testing URL for testing.
+     */
+    @VisibleForTesting
+    fun setTestingNeuroIDDevURL()
+
+    /**
+     * Set a user id.
+     */
+    fun setUserID(userID: String): Boolean
+
+    /**
+     * Return the currently set user id.
+     */
+    fun getUserId(): String
+    /**
+     * Return the currently set user id.
+     */
+    fun getUserID(): String
+    /**
+     * Return the currently set registered user id.
+     */
+    fun getRegisteredUserID(): String
+
+    /**
+     * Set a registered user id.
+     */
+    fun setRegisteredUserID(registeredUserId: String): Boolean
+
+    /**
+     * Set a screen name for the current page in a session.
+     */
+    fun setScreenName(screen: String): Boolean
+
+    /**
+     * Return the currently set screen name for the current page in a session.
+     */
+    fun getScreenName(): String
+
+    /**
+     * Register page targets. This will send events targets to the server for the current page.
+     */
+    fun registerPageTargets(activity: Activity)
+
+    /**
+     * Exclude the event that is specified. Used for testing purposes.
+     */
+    @VisibleForTesting
+    fun excludeViewByTestID(id: String)
+
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("setEnvironment is deprecated and no longer required")
+    fun setEnvironment(environment: String)
+
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("setEnvironmentProduction is deprecated and no longer required")
+    fun setEnvironmentProduction(prod: Boolean)
+
+    /**
+     * Return the current environment set for operation (live/test).
+     */
+    fun getEnvironment(): String
+
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("setSiteId is deprecated and no longer required")
+    fun setSiteId(siteId: String)
+
+    /**
+     * Return the current sessionId.
+     */
+    fun getSessionId(): String
+
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("formSubmit is deprecated and no longer required")
+    fun formSubmit()
+
+    /**
+     * Deprecated, do not use!
+     */
+    @Deprecated("formSubmitSuccess is deprecated and no longer required")
+    fun formSubmitSuccess()
+
+    /**
+     * Deprecated, do not use
+     */
+    @Deprecated("formSubmitFailure is deprecated and no longer required")
+    fun formSubmitFailure()
+
+    /**
+     * Start the SDK and start a new session and use the userID as the sessionID. Return true if
+     * that indicates that the SDK is started. Returns false if the SDK is stopped. In the case of
+     * a false return, check that the client key is set properly in the NeuroID.Builder() call.
+     */
+    fun start(): Boolean
+
+    /**
+     * Stop the SDK and close the current session. Return true to indicate that
+     * the SDK is stopped.
+     */
+    fun stop(): Boolean
+
+    /**
+     * Stop the SDK and closes the current session.
+     */
+    fun closeSession()
```

Comment:
I don't know that we want this exposed externally since they should be calling `stop` or `stopSession`

Reply:
removed